docs/src/tutorials/tutorial_1.md

```diff
@@ -64,7 +64,7 @@ end
 ```
 
 > **NOTE:** One may wonder, "why is `args...` needed in the `ground` function?"
-> Each user-defined function gets three arguments `video`, `object`, and `frame` provided by Javis
+> Each user-defined function gets three arguments `video`, `object`, and `frame` provided by Javis  
```

Comment:
I think I would also add a `.` at the end of the sentence.

Reply:
Thanks, I've added that now.